### PR TITLE
Add HEARTBEAT_DT question

### PR DESCRIPTION
**Pull request by sdrabenh**
Add an additional input question to `gcm_setup` asking the user for the model `HEARTBEAT_DT`. Override with 450 for `BACM_1M` cases.

### CODE REVIEW

File: gcm_setup [hunk 1]
```diff
@@ -1439,6 +1439,36 @@ endif
...
+echo "   ${C2}2. Type an integer (in seconds) to overide the default value ${CN}"
+
+set    HEARTBEAT =  $<
+if(  .$HEARTBEAT == . ) then
+   set HEARTBEAT =  $DT
+else
+   set HEARTBEAT = `echo $HEARTBEAT | awk '{print ($0+0)}'`
+   if ( ! $HEARTBEAT > 0 ) then
+      echo
+      echo "${C1}HEARTBEAT${CN} must be a valid integer using one of the options below!"
```
Comment (mathomp4) [inline gcm_setup:1464]: What is one of the options below?

**Comment by mathomp4** [comment]
A question for @wmputman: should we make this "resolution dependent"? I mean, `DT=450` is the default (currently) for C360. For lower resolutions it ranges from 600 (C180) to 1800 seconds (C12), and for higher resolutions the `DT` is lower.

File: gcm_setup (hunk 1 above)
Comment (sdrabenh) [reply gcm_setup:1464]: the next line goes to the label and echoes the original question and options (1. and 2.) again

Comment (mathomp4) [reply gcm_setup:1464]: Ohh. okay. I was thinking there was a list of valid DTs!

**Comment by mathomp4** [comment]
I've updated the CI with my new `create_expt.py` script which should support the new heartbeat question. We'll see if the run works.

**Comment by mathomp4** [comment]
Sigh. Found more bugs. Update incoming...

**Review by mathomp4 — APPROVED** [review]
I think the CI is working now. So I'll approve. I also added the heartbeat chunk to the other setups.

**Pull request merged**